packages/react/src/progress-indicator/docs/overview.mdx

```diff
@@ -144,6 +144,24 @@ If an item does not specify an `href` attribute, a `button` element will be rend
 />
 ```
 
+## Custom active item
```

Comment:
I think this content should be before Buttons, because it's more important and directly related to the Status

Reply:
Moved